functional_tests/testdata/expected_eks_values/expected_cluster_receiver.yaml

```diff
@@ -1150,6 +1174,42 @@ resourceMetrics:
                       value:
                         stringValue: k8scluster
                   timeUnixNano: "1000000"
+                - asInt: "1"
+                  attributes:
+                    - key: container.id
+                      value:
+                        stringValue: 38809b5e5212eddddeec2d4d387e2f27ff754e0c3f67cc9a9ffb89db72601480
+                    - key: container.image.name
+                      value:
+                        stringValue: quay.io/splunko11ytest/dotnet_test
+                    - key: container.image.tag
+                      value:
+                        stringValue: latest
+                    - key: k8s.cluster.name
+                      value:
+                        stringValue: rotel-eks
+                    - key: k8s.container.name
+                      value:
+                        stringValue: dotnet-test
+                    - key: k8s.namespace.name
+                      value:
+                        stringValue: default
+                    - key: k8s.node.name
+                      value:
+                        stringValue: kind-control-plane
+                    - key: k8s.pod.name
+                      value:
+                        stringValue: dotnet-test-57564b7dc9-pf2bk
+                    - key: k8s.pod.uid
+                      value:
+                        stringValue: b660efd2-e961-488a-af7a-2161f27565bb
```

Comment:
Is this id always the same?

Reply:
In our expected results content used for test validation, ID values like these are filler strings (can be repeated), as they're ignored by the validation logic. In a real environment, these IDs are randomly generated and unique.